src/main/java/uk/gov/hmcts/cmc/claimstore/courtfinder/models/CourtDetails.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.cmc.claimstore.courtfinder.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+public class CourtDetails {
+    @Getter private String name;
```

Comment:
I would suggest adding `@Getter` on the class level instead of every field
```suggestion
    private String name;
```